news/2022-11-05-new-featured-artist-massive-new-krew.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "New Featured Artist: Massive New Krew"
+date: 2022-11-05 11:00:00 +0000
+---
+
+**Massive New Krew** blast the doors into our Featured Artist collection.
+
+![](https://assets.ppy.sh/artists/318/header.jpg)
+
+Pounding juicy bass since 2008, **Massive New Krew** is a duo of electronic producers who've established themselves as hardstyle icons. They earned their reputation by performing as a club DJ at major events throughout Japan and shattering ears in rhythm games like *beatmania IIDX*, *CHUNITHM*, *Groove Coaster*, and *Taiko no Tatsujin*!
```

Comment:
```suggestion
Pounding juicy bass since 2008, **Massive New Krew** is a duo of electronic producers who've established themselves as hardstyle icons. They earned their reputation by performing as club DJs at major events throughout Japan and shattering ears in rhythm games like *beatmania IIDX*, *CHUNITHM*, *Groove Coaster*, and *Taiko no Tatsujin*!
```